Source/WebCore/platform/graphics/gstreamer/GStreamerRegistryScanner.cpp

```diff
@@ -660,6 +667,41 @@ MediaPlayerEnums::SupportsType GStreamerRegistryScanner::isContentTypeSupported(
 
     const auto& codecs = contentType.codecs();
 
+#if ENABLE(ENCRYPTED_MEDIA)
+    String cryptoblockformat = contentType.parameter("cryptoblockformat"_s);
+    if (!cryptoblockformat.isEmpty()) {
+        OptionSet<ElementFactories::Type> factoryTypes;
+        factoryTypes.add(ElementFactories::Type::Decryptor);
+        auto factories = ElementFactories(factoryTypes);
+
+        // https://developers.google.com/youtube/devices/living-room/certification-requirements
+        // section 15.1.3: The isTypeSupported method MUST support the cryptoblockformat mime-type
+        // parameter, and MUST support subsample as a parameter value.
+        if (cryptoblockformat != "subsample"_s)
+            return SupportsType::IsNotSupported;
+
+        const char* encryptionScheme = containerType.endsWith("webm"_s) ? "application/x-webm-enc" : "application/x-cenc";
```

Comment:
Considering what we are commenting, I'd just bail out with `IsNotSupported` if container is not WebM then continue with the rest of the checks.